.travis.yml

```diff
@@ -51,3 +51,13 @@ script:
 
 after_success:
   - if [[ "$WITH_COVERAGE" == "true" ]]; then php composer/bin/test-reporter; fi
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_API_KEY
```

Comment:
I think if we use this syntax:

```
deploy:
  provider: releases
  api_key:
    secure: YOUR_API_KEY_ENCRYPTED
```

then we can feel confident that it shouldn't be logged by travis, because travis will know it's a secure thing.